### ISSUE TITLE
Question - how to set up a recording form for a set of families

### ISSUE THREAD

**Issue by kitenetter** [BiologicalRecordsCentre/iRecord#846]
I'm trying to set up a recording form for sawflies. Sawflies fall within the Hymenoptera, and do not have their own informal taxon group in the UKSI, so we are only able to define them by listing the 12 relevant families.

Is it possible to set filters within the form itself to restrict taxon names to just the taxa in those 12 families?

If not, do I need to set up a new species list in the warehouse? If so, is there any documentation on how to use a subset of UKSI to create a new species list?

The draft form is here:
https://www.brc.ac.uk/irecord/node/7998

**Comment by johnvanbreda** [comment]
@kitenetter In the warehouse, you can go to the child lists tab for the main list (http://warehouse1.indicia.org.uk/index.php/taxon_list/15). I haven't used it for a while but the Taxa tab of any child lists you add will have tools for cloning taxa from the main list into your list, including a way of bulk-pasting names. 

I don't think the forms can currently filter the search on anything other than individual taxa or taxon groups.

**Comment by kitenetter** [comment]
Thanks John. 

For reference, I used the UKSI sandbox to download an excel list of all taxa within Symphyta:
http://nbn-sd-dev.nhm.ac.uk/taxonbrowser.php?txtSearch=symphyta

From that list I excluded intermediate taxon ranks such as subfamilies and subgenera; I excluded synonyms; I excluded ill-formed and redundant names; I excluded English names. That left me with 675 names.

The warehouse import process says:
"Paste a list of taxon names or external keys into the box above then click Add to add them all"

So I tried pasting in the list of TVKs, but all were rejected with the message "NBNSYS0000159623 could not be found in the parent list" (and so on for each TVK).

I then tried with the name strings, and this time 545 of the 675 were imported successfully. 122 "could not be found in the parent list" and 8 were "found but could not be used to identify a unique taxon in the parent list".

At least some of the mismatches will be the result of the UKSI being very up-to-date and including name changes that iRecord is not yet aware of.

This list of 545 names is probably sufficient for the survey at the moment, so no further action needed, unless @johnvanbreda can tell me how to make use of the TVKs/external keys. 

Closing.
